samples/browser-esm-vite-vue3/src/components/Editor.vue

```diff
@@ -0,0 +1,29 @@
+<script setup>
+import { ref, onMounted, onUnmounted, toRaw } from 'vue';
+import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
+
+const editorRef = ref(null);
+const editorInstance = ref(null);
+
+onMounted(() => {
+	if (editorRef.value && !editorInstance.value) {
+		editorInstance.value = monaco.editor.create(editorRef.value, {
+			value: ['function x() {', '\tconsole.log("Hello world!");', '}'].join('\n'),
+			language: 'javascript'
+		});
+	}
+});
+
```

Comment:
Would be nice to add a `save` example, witch in my case took me some work to find out how I should actually do it.

```ts
    const save = () => {
        const code = monaco.editor.getEditors()[0].getValue();
    };
```

Reply:
If you want to do this you must make sure the editor is instantiated, otherwise `getEditor()` may return `undefined`, which will cause an error.

Reply:
@recallwei thanks. My suggestion is to add to this PR the example, so will be easier to other devs.

Cheers